sql/moz-fx-data-shared-prod/monitoring_derived/jobs_by_organization_v1/query.py

```diff
@@ -12,6 +12,11 @@
     "moz-fx-data-shared-prod",
     "moz-fx-data-marketing-prod",
     "moz-fx-data-bq-data-science",
+    "moz-fx-glam-prod",
+    "moz-fx-glam-nonprod",
+    "moz-fx-data-sumo-prod",
```

Comment:
You managed to pick some of the hard hitters, these projects are [restricted access](https://github.com/mozilla-it/global-platform-admin/blob/main/bootstrap/tf/restricted.tf#L2) and so metadata about them is subject to additional scrutiny. That said, I have spent at least some amount of time in https://mozilla-hub.atlassian.net/browse/DSRE-1596 and https://mozilla-hub.atlassian.net/browse/DSRE-1179 trying to make these projects no longer exist, and I expect the work to finalize that to pick up (possibly with DE involvement) as part of the transition work.

It's expected in the end state that those projects be superseded by configuration in `moz-fx-sumo-nonprod` and `moz-fx-sumo-prod`  which are standard GCPv2 projects.